Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -436,50 +436,61 @@ inline void SeedFinder<external_spacepoint_t, platform_t>::filterCandidates(
     const InternalSpacePoint<external_spacepoint_t>& spM,
     const Acts::SeedFinderOptions& options, SeedFilterState& seedFilterState,
     SeedingState& state) const {
-  float rM = spM.radius();
-  float cosPhiM = spM.x() / rM;
-  float sinPhiM = spM.y() / rM;
-  float varianceRM = spM.varianceR();
-  float varianceZM = spM.varianceZ();
+  const float& rM = spM.radius();
+  const float cosPhiM = spM.x() / rM;
+  const float sinPhiM = spM.y() / rM;
+  const float& varianceRM = spM.varianceR();
+  const float& varianceZM = spM.varianceZ();
```

Comment:
The references will require more storage (64bit) than the floats (32bit), so I think the copies are actually cheaper.

Reply:
I believe references and floats are both 4 bytes, no? same for unsigned long int

Reply:
ok... it's 4 bytes on 32-bit, 8 bytes on 64-bit